book/CH05_htmxPatterns.adoc

```diff
@@ -1032,3 +1032,34 @@ And, again, we are doing all this within the original, RESTful model of the web.
 behavior, we are still exchanging hypermedia with the server, with no JSON API response to be seen.
 
 As the web was designed.
+
+
+[.design-note]
+.Component Soup?
```

Comment:
The notes on "component soup" and "web components" have some overlapping themes, but don't quite mesh together. I might rewrite "component soup" as "framework soup" and move some bits from it to webcomponents